<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="com.kodilla.exception.io.secondChallange.SecondChallengeTestSuite" tests="2" skipped="0" failures="0" errors="0" timestamp="2023-03-24T14:24:00" hostname="DESKTOP-Q0KKPO1" time="0.01">
  <properties/>
  <testcase name="testProbablyWillThrowException()" classname="com.kodilla.exception.io.secondChallange.SecondChallengeTestSuite" time="0.004"/>
  <testcase name="willThrowExceptionCases()" classname="com.kodilla.exception.io.secondChallange.SecondChallengeTestSuite" time="0.004"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
